### PR TITLE
Build initial 'Notable Person' page

### LINKED ISSUE #58: Build 'Notable Person' Page
At the moment, we haven't decided on how other pages (search, create, edit) will interact with the 'Notable Person' page. 

However, we know for sure that we'll have a 'Notable Person' page and we have an early vision of this page which is clearly described in the [product description](https://github.com/hollowverse/hollowverse/wiki/Product-Description). Also an early [mock-up](https://github.com/hollowverse/hollowverse/blob/master/design-docs/mockups/_outputs/MobilePortrait_v01_InAction.jpg) is available. 

Based on this information, we'll start building the 'Notable Person' page and track the initial progress with this issue. The first thing to do is to flesh out the components that belong to this page with some mock data.

Moving forward, we'll start thinking about how to integrate this page.

- [x] Build initial Notable Person page
- [x] Implement Events
- [x] Connect to Firebase backend

### PR DESCRIPTION
**Pull request by levsthings**
- [x] Pull in the old work done for Notable Person page.
- [x] Rewrite existing styles in aphrodite.
- [x] Finish Notable Person page v1.
- [x] Eliminate Bulma and old Global styles.

### CODE REVIEW

File: client/src/pages/notablePerson/notablePerson.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import {css} from 'aphrodite/no-important'
+import {RouteComponentProps} from 'react-router-dom'
+import {common} from '../../common.styles'
+import {styles} from './notablePerson.styles'
+
+interface Props {
+  notablePersonName: string
+  notablePersonPictureUrl: string
+  notablePersonLabels: [string]
```
Comment (wholesomedev) [inline client/src/pages/notablePerson/notablePerson.tsx]: Is this an alternative way to type an array of strings? The way I know is `string[]`

Comment (levsthings) [reply client/src/pages/notablePerson/notablePerson.tsx]: It's actually the tuple syntax, I had a number in there as well but removed it afterwards. But string stayed :)  `string[]` is the way to go for the current data set.

Comment (wholesomedev) [reply client/src/pages/notablePerson/notablePerson.tsx]: Ah, TIL

File: client/src/pages/notablePerson/notablePerson.styles.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+export const styles = StyleSheet.create({
+  notablePersonTitleContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: '1.5rem',
+    margin: '0.8rem 0.8rem',
+  },
+  notablePersonPhoto: {
+    borderRadius: '80px',
+    maxWidth: '150px',
```
Comment (wholesomedev) [inline client/src/pages/notablePerson/notablePerson.styles.ts]: Isn't it better for this to be a percentage?

File: client/src/pages/notablePerson/notablePerson.styles.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+export const styles = StyleSheet.create({
+  notablePersonTitleContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: '1.5rem',
+    margin: '0.8rem 0.8rem',
+  },
+  notablePersonPhoto: {
+    borderRadius: '50%',
+    maxWidth: '150px',
```
Comment (wholesomedev) [inline client/src/pages/notablePerson/notablePerson.styles.ts]: Should this be a percentage?

File: client/src/pages/notablePerson/notablePerson.styles.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import {StyleSheet} from 'aphrodite/no-important'
+
+export const styles = StyleSheet.create({
+  notablePersonTitleContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: '1.5rem',
+    margin: '0.8rem 0.8rem',
```
Comment (wholesomedev) [inline client/src/pages/notablePerson/notablePerson.styles.ts]: Will `margin` overwrite `marginTop`?

File: client/src/pages/notablePerson/notablePerson.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+type ComponentProps = Props & RouteComponentProps<any>
+
+class NotablePersonClass extends React.Component<ComponentProps, undefined> {
+  render() {
+    return (
+      <div className={css(common.page)}>
+        <div className={css(styles.notablePersonTitleContainer)}>
+          <img className={css(styles.notablePersonPhoto)} src={dummyData.notablePersonPictureUrl}/>
+          <div className={css(styles.notablePersonText)}>
+            <h1 className={css(styles.notablePersonTitle)}>Religion, Politics, and ideas of...</h1>
```
Comment (wholesomedev) [inline client/src/pages/notablePerson/notablePerson.tsx]: Can we sentence-case this?

**Review by wholesomedev — CHANGES_REQUESTED** [review]

File: client/src/pages/notablePerson/notablePerson.styles.ts (hunk 4 above)
Comment (levsthings) [reply client/src/pages/notablePerson/notablePerson.styles.ts]: Good catch, it was a leftover property from the early implementation.

**Review by wholesomedev — APPROVED** [review]

**Comment by wholesomedev** [comment]
LGTM! I've updated the PR message to remove the "Close" keyword because I think we need to keep #58 open until we have fully completed the Notable Person page, including having it work against an actual backend.

**Pull request merged**